README.md

```diff
@@ -167,6 +167,17 @@ fastify.route({
 | `csrfOpts` |  The csrf options. See  [csrf](https://github.com/pillarjs/csrf).     |
 
 
+## Securing the secret
+
+The `secret` shown in above code is strictly just a sample. In all cases, you'd need to make sure that the `secret` is:
+- **Never** hard-coded in the code or `.env` files or anywhere in the repository
+- Stored in some external services like KMS, Vault or something similar
+- Read at run-time and supplied in this option
+- Long enough
+- Truly random sequence of characters
```

Comment:
Is there a module or service we can recommend to generate this?

Reply:
Added hyperid & crypto-random-string

Reply:
I don't think hyperid can be considered a truly random char sequence generator in the sense we intend here. It does generate unique ids, just not random

Reply:
I just thought a generated hyperid string is good enough for a cookie secret, but yes, technically you're right.

Reply:
In this context it isn't. If you can guess what id is generated and you get hold of an encrypted cookie, you can decrypt it. In this context we need cryptographically secure secrets.